Artesian/Artesian.SDK/Factory/MarketDataMetadata.cs

```diff
@@ -89,5 +93,47 @@ internal MarketDataMetadata(MarketDataEntity.Output output)
         /// The time the market data has been created
         /// </summary>
         public Instant Created => _output.Created;
+        /// <summary>
```

Comment:
nope. expose the Cfg as nested object, do not flatten its properties.

the hierarchy must be preserved.

I'm not sure is a 'metadata' property. Derived CFG looks to me more a Registry property as it defines the MarketData itself (same as MarketDataType).

Place it alongside the MarketDataType

{
   MarketDataType,
   DerivedCfg {
      Algo
      Dependants
   }
}